packages/webpack/src/plugins/webpack-nx-build-coordination-plugin.ts

```diff
@@ -42,46 +54,46 @@ export class WebpackNxBuildCoordinationPlugin {
       await sleep(50);
     }
     this.currentlyRunning = 'nx-build';
-    try {
-      execSync(this.buildCmd, { stdio: [0, 1, 2] });
-      // eslint-disable-next-line no-empty
-    } catch (e) {}
-    this.currentlyRunning = 'none';
+    return new Promise<void>((res) => {
+      try {
+        const cp = exec(this.buildCmd);
+
+        cp.stdout.pipe(process.stdout);
+        cp.stderr.pipe(process.stderr);
+        cp.on('exit', () => {
+          res();
+        });
+        cp.on('error', () => {
+          res();
```

Comment:
Should this be `rej`

Reply:
Probably, but the original logic noop'd for errors so this basically does the same.

Reply:
No problem!